src/java.base/share/classes/jdk/internal/classfile/impl/StackCounter.java

```diff
@@ -56,15 +56,15 @@ static StackCounter of(DirectCodeBuilder dcb, BufWriterImpl buf) {
     private int stack, maxStack, maxLocals, rets;
 
     private final RawBytecodeHelper bcs;
-    private final String methodName;
-    private final MethodTypeDesc methodDesc;
+    private final Utf8Entry methodName;
+    private final Utf8Entry methodDesc;
     private final SplitConstantPool cp;
-    private final LinkedHashMap<Integer, Integer> map;
+    private final LinkedList<Target> targets;
```

Comment:
Wouldn't it be better to use `ArrayDeque` instead of `LinkedList`?

Reply:
Good point, I'll change it, thanks.